### ISSUE TITLE
[Task] Let's make Assigned Member  Mandatory filed while creating the Task

### ISSUE THREAD

**Issue by drasticdpk** [topcoder-platform/work-manager#1145]
**Reference** 
![image](https://user-images.githubusercontent.com/25297859/117282065-0f449100-ae84-11eb-82a0-10b3aeb7d2f1.png)

**Comment by coderhacker** [comment]
Is is not fixed on  Production: 
![image](https://user-images.githubusercontent.com/62358241/119330352-be1a0700-bca3-11eb-9a32-0e534284331c.png)

**Comment by vikasrohit** [comment]
I don't think we attempted to fix it @lakshmiathreya, is there any release after 0.8.0 for which we are doing testing round?
